source/EngineInterface/SerializerImpl.cpp

```diff
@@ -150,11 +150,9 @@ namespace boost {
         template<class Archive>
 		inline void serialize(Archive & ar, SimulationParameters& data, const unsigned int /*version*/)
```

Comment:
This method is not used anymore (I haven't cleaned the code...). Simulation parameters are now serialized to JSON format.